omnibus/Gemfile

```diff
@@ -1,7 +1,7 @@
 source "https://rubygems.org"
 
-gem "omnibus", git: "https://github.com/chef/omnibus.git", branch: "master"
-gem "omnibus-software", git: "https://github.com/chef/omnibus-software.git", branch: "master"
+gem 'omnibus', github: ENV.fetch('OMNIBUS_GITHUB_REPO', 'chef/omnibus'), branch: ENV.fetch('OMNIBUS_GITHUB_BRANCH', 'master')
```

Comment:
We'll need to confirm if this breaks dependabot. They tend to just fail if you put any logic into your gemfile for security reasons.

Reply:
It doesn't appear to, at least with the native GitHub-based version: https://github.com/chef/omnibus-harmony/network/dependencies

Reply:
The logs from the scan itself seem to indicate that dependabot has no trouble with this particular logic.

Reply:
Nice